openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -87,16 +85,12 @@ def _get_client_id(self, request):
             return request.POST.get('client_id')
 
 
-class AccessTokenView(RatelimitMixin, _DispatchingView):
+class AccessTokenView(_DispatchingView):
```

Comment:
```suggestion
# TODO: Edraak-specific: Temporarily removing the RatelimitMixin mixin to avoid Ratelimit-related issues.
class AccessTokenView(_DispatchingView):
```